OWNERS

```diff
@@ -8,4 +8,7 @@ approvers:
 
 reviewers:
   - cluster-api-addon-helm-maintainers
-  - cluster-api-addon-helm-reviewers
\ No newline at end of file
+  - cluster-api-addon-helm-reviewers
+
+emeritus_approvers:
+  - CecileRobertMichon
```

Comment:
Could you add a new line?

Reply:
There is a newline there now but wasn't before AFAICT.